[Snyk] Fix for 2 vulnerabilities

mod opened the pull request:
![snyk-top-banner](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/pr-banner-default.svg)

### Snyk has created this PR to fix 2 vulnerabilities in the rubygems dependencies of this project.

#### Snyk changed the following file(s):

- `Gemfile`



<details>
<summary>⚠️ <b>Warning</b></summary>

```
Failed to update the Gemfile.lock, please update manually before merging.
```

</details>



#### Vulnerabilities that will be fixed with an upgrade:

|  | Issue | Score | 
:-------------------------:|:-------------------------|:-------------------------
![high severity](https://res.cloudinary.com/snyk/image/upload/w_20,h_20/v1561977819/icon/h.png 'high severity') | Allocation of Resources Without Limits or Throttling <br/>[SNYK-RUBY-RACK-13052974](https://snyk.io/vuln/SNYK-RUBY-RACK-13052974) | &nbsp;&nbsp;**696**&nbsp;&nbsp; 
![medium severity](https://res.cloudinary.com/snyk/image/upload/w_20,h_20/v1561977819/icon/m.png 'medium severity') | Regular Expression Denial of Service (ReDoS) <br/>[SNYK-RUBY-REXML-13110060](https://snyk.io/vuln/SNYK-RUBY-REXML-13110060) | &nbsp;&nbsp;**631**&nbsp;&nbsp; 




---

> [!IMPORTANT]
>
> - Check the changes in this PR to ensure they won't cause issues with your project.
> - Max score is 1000. Note that the real score may have changed since the PR was raised.
> - This PR was automatically created by Snyk using the credentials of a real user.

---

**Note:** _You are seeing this because you or someone else with access to this repository has authorized Snyk to open fix PRs._

For more information: <img src="https://api.segment.io/v1/pixel/track?data=eyJ3cml0ZUtleSI6InJyWmxZcEdHY2RyTHZsb0lYd0dUcVg4WkFRTnNCOUEwIiwiYW5vbnltb3VzSWQiOiIxMzA5ZjNjMS0zMTNmLTRlMzItODcwZi1jYjkxYzIxYWE4OTgiLCJldmVudCI6IlBSIHZpZXdlZCIsInByb3BlcnRpZXMiOnsicHJJZCI6IjEzMDlmM2MxLTMxM2YtNGUzMi04NzBmLWNiOTFjMjFhYTg5OCJ9fQ==" width="0" height="0"/>
🧐 [View latest project report](https://app.snyk.io/org/mod/project/478f2032-55a5-4895-864a-35b37f77eaa9?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr)
📜 [Customise PR templates](https://docs.snyk.io/scan-using-snyk/pull-requests/snyk-fix-pull-or-merge-requests/customize-pr-templates?utm_source=github&utm_content=fix-pr-template)
🛠 [Adjust project settings](https://app.snyk.io/org/mod/project/478f2032-55a5-4895-864a-35b37f77eaa9?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr/settings)
📚 [Read about Snyk's upgrade logic](https://docs.snyk.io/scan-with-snyk/snyk-open-source/manage-vulnerabilities/upgrade-package-versions-to-fix-vulnerabilities?utm_source=github&utm_content=fix-pr-template)

---

**Learn how to fix vulnerabilities with free interactive lessons:**

🦉 [Allocation of Resources Without Limits or Throttling](https://learn.snyk.io/lesson/no-rate-limiting/?loc&#x3D;fix-pr)
🦉 [Regular Expression Denial of Service (ReDoS)](https://learn.snyk.io/lesson/redos/?loc&#x3D;fix-pr)

[//]: # 'snyk:metadata:{"breakingChangeRiskLevel":null,"FF_showPullRequestBreakingChanges":null,"FF_showPullRequestBreakingChangesWebSearch":null,"customTemplate":{"variablesUsed":[],"fieldsUsed":[]},"dependencies":[{"name":"factory_bot_rails","from":"5.2.0","to":"6.0.0"},{"name":"faye","from":"1.4.0","to":"1.4.1"},{"name":"grape","from":"1.3.3","to":"1.4.0"},{"name":"grape-swagger","from":"0.30.1","to":"0.31.0"},{"name":"grape-swagger-entity","from":"0.2.5","to":"0.3.0"},{"name":"grape_logging","from":"1.8.4","to":"2.0.0"},{"name":"jwt-rack","from":"0.1.0","to":"0.1.1"},{"name":"peatio","from":"2.6.2","to":"2.6.3"},{"name":"peatio-bitcoincash","from":"2.6.1","to":"3.1.0"},{"name":"peatio-bitgo","from":"2.6.3","to":"2.6.4"},{"name":"peatio-dash","from":"2.6.1","to":"3.1.0"},{"name":"peatio-electrum","from":"2.6.2","to":"2.6.3"},{"name":"peatio-litecoin","from":"2.6.1","to":"3.1.0"},{"name":"peatio-ripple","from":"2.6.1","to":"3.1.0"},{"name":"rack-attack","from":"5.4.2","to":"6.0.0"},{"name":"rack-cors","from":"1.0.6","to":"1.1.0"},{"name":"rails","from":"5.2.4.4","to":"5.2.4.5"},{"name":"rspec-rails","from":"3.9.0","to":"3.9.1"},{"name":"rubocop-rspec","from":"1.43.2","to":"2.0.0"},{"name":"vault-rails","from":"0.6.0","to":"0.7.0"}],"env":"prod","issuesToFix":["SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-RACK-13052974","SNYK-RUBY-REXML-13110060"],"prId":"1309f3c1-313f-4e32-870f-cb91c21aa898","prPublicId":"1309f3c1-313f-4e32-870f-cb91c21aa898","packageManager":"rubygems","priorityScoreList":[696,631],"projectPublicId":"478f2032-55a5-4895-864a-35b37f77eaa9","projectUrl":"https://app.snyk.io/org/mod/project/478f2032-55a5-4895-864a-35b37f77eaa9?utm_source=github&utm_medium=referral&page=fix-pr","prType":"fix","templateFieldSources":{"branchName":"default","commitMessage":"default","description":"default","title":"default"},"templateVariants":["pr-warning-shown","priorityScore"],"type":"auto","upgrade":["SNYK-RUBY-RACK-13052974","SNYK-RUBY-REXML-13110060"],"vulns":["SNYK-RUBY-RACK-13052974","SNYK-RUBY-REXML-13110060"],"patch":[],"isBreakingChange":true,"remediationStrategy":"vuln"}'

yivo commented:
@vpetrusenko ping

vpetrusenko commented:
@yivo hello, with this changes https://github.com/rubykube/peatio/pull/59 i dont have this spec fail anymore, so i think this pull request should be closed, also could you please assign to me another task?

yivo commented:
@vpetrusenko Got it! Check https://github.com/rubykube/peatio/labels/assigned%3A%20Valera

snyk-bot committed:
fix: Gemfile to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-RUBY-RACK-13052974
- https://snyk.io/vuln/SNYK-RUBY-REXML-13110060
